specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -5227,22 +5725,172 @@
       ],
       "x-ms-enum": {
         "name": "SplitSkillLanguage",
-        "modelAsString": false
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "da",
+            "description": "Danish"
+          },
+          {
+            "value": "de",
+            "description": "German"
+          },
+          {
+            "value": "en",
+            "description": "English"
+          },
+          {
+            "value": "es",
+            "description": "Spanish"
+          },
+          {
+            "value": "fi",
+            "description": "Finnish"
+          },
+          {
+            "value": "fr",
+            "description": "French"
+          },
+          {
+            "value": "it",
+            "description": "Italian"
+          },
+          {
+            "value": "ko",
+            "description": "Korean"
+          },
+          {
+            "value": "pt",
+            "description": "Portuguese"
+          }
+        ]
       },
       "description": "The language codes supported for input text by SplitSkill."
     },
     "EntityRecognitionSkillLanguage": {
       "type": "string",
       "enum": [
-        "de",
+        "ar",
```

Comment:
Do you still need this list when using extensible enums? It seems to be omitted elsewhere.

Reply:
I looked at all instances in our spec - we specify both enum array (and when required) the values under x-ms-enum.

Swagger documentation, recommends we use both, in case there are clients that might not know how to handle x-ms-enum (https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-enum)

>

Reply:
Got it. I was fooled by the diff. :)